packages/flutter_tools/test/general.shard/runner/flutter_command_test.dart

```diff
@@ -1208,6 +1209,148 @@ void main() {
         ProcessManager: () => FakeProcessManager.any(),
       });
     });
+
+    group('Flutter version', () {
+      late _TestDeviceManager testDeviceManager;
+      late Logger logger;
+      late FileSystem fileSystem;
+
+      const List<String> flutterVersionDartDefines = <String>[
+        'FLUTTER_VERSION',
+        'FLUTTER_CHANNEL',
+        'FLUTTER_GIT_URL',
+        'FLUTTER_FRAMEWORK_REVISION',
+        'FLUTTER_ENGINE_REVISION',
+        'FLUTTER_DART_VERSION',
+      ];
+
+      setUp(() {
+        logger = BufferLogger.test();
+        testDeviceManager = _TestDeviceManager(logger: logger);
+        fileSystem = MemoryFileSystem.test();
+      });
```

Comment:
please avoid using shared variables in new tests, see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#make-each-test-entirely-self-contained

Reply:
Done